data/json/items/tool/woodworking.json

```
@@ -280,5 +280,48 @@
     "color": "light_gray",
     "qualities": [ [ "CHISEL", 1 ] ],
     "flags": [ "BELT_CLIP" ]
+  },
+  {
+    "id": "tomahawk_primitive",
+    "type": "GENERIC",
+    "category": "weapons",
+    "weapon_category": [ "1H_AXES" ],
+    "symbol": ";",
+    "color": "light_gray",
+    "name": { "str": "stone tomahawk" },
+    "description": "A lightweight one-handed combat hatchet typically made of stone or bone, originally used by the Algonquian people.  It makes for a half decent tool and weapon.",
+    "price_postapoc": 100,
+    "material": [ "wood", "stone" ],
+    "techniques": [ "WBLOCK_1" ],
+    "volume": "1750 ml",
+    "weight": "2800 g",
+    "looks_like": "hatchet",
+    "bashing": 5,
+    "cutting": 12,
+    "to_hit": 1,
+    "flags": [ "DURABLE_MELEE", "SHEATH_AXE" ],
+    "qualities": [ [ "CUT", 1 ], [ "BUTCHER", 10 ] ]
+  },
+  {
+    "id": "tomahawk_copper",
+    "type": "GENERIC",
+    "category": "weapons",
+    "weapon_category": [ "1H_AXES" ],
+    "symbol": ";",
+    "color": "light_red",
+    "name": { "str": "copper tomahawk" },
+    "description": "A lightweight one-handed combat hatchet made using basic copper;  slightly better than stone, though not as historically accurate.",
```

Removing the extra space since semicolon would be an option but could do separate sentence instead, whichever?
```suggestion
    "description": "A lightweight one-handed combat hatchet made using basic copper.  Slightly better than stone, though not as historically accurate.",
```